apps-rendering/src/date.ts

```diff
@@ -125,7 +125,7 @@ const month = (date: Date): string => months[date.getUTCMonth()];
 
 const fullMonth = (date: Date): string => fullMonths[date.getUTCMonth()];
 
-const padZero = (n: number): string => (n < 10 ? `0${n}` : n.toString());
+const padZero = (n: number): string => String(n).padStart(2, '0');
```

Comment:
Maybe a hot take but I prefer `n.toString().padStart(2, '0')` than `String(n).padStart(2, '0')`!

Reply:
oh interesting! I don't think I have a strong preference for either, but your comment did send me down a rabbit hole to investigate the differences, so thank you for that!

What I learned:

- `n.toString()` will throw a `TypeError` if `n` is `undefined` - makes sense, not really a concern in this case. Whereas `String(n)` will always give you a `string` back, even if the input is `undefined` unless `n` is missing `toString` or `valueOf`
- `String(n)` will call `n.toString()` if it exists

I enjoyed reading this article about [Object to Primitive conversions](http://www.adequatelygood.com/Object-to-Primitive-Conversions-in-JavaScript.html)